src/sql/workbench/contrib/notebook/browser/calloutDialog/media/calloutDialog.css

```diff
@@ -0,0 +1,116 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.modal.callout-dialog {
```

Comment:
I'd suggest having a separate CSS file for each time of callout dialog that contains just the styles relevant for that dialog, and then this one would just have the general common styles.

Reply:
I cleaned up modal.css, then created calloutDialog.css with only callout generic. Then created imageCalloutDialog and linkCalloutDialog and keep in these two only CSS specific to each. (linkCalloutDialog.css is empty but I want it there in case I have to adjust it.)